crates/node-builder/src/builder.rs

```diff
@@ -1062,7 +1123,7 @@ where
     }
 
     /// Installs an ExEx (Execution Extension) in the node.
-    pub fn install_exex<F, R, E>(mut self, exex: F) -> Self
+    pub fn install_exex<F, R, E>(mut self, exex_id: String, exex: F) -> Self
```

Comment:
```suggestion
    pub fn install_exex<F, R, E>(mut self, exex_id: AsRef<str>, exex: F) -> Self
```

so we can pass any string